contexts/credentials/v2

```diff
@@ -273,7 +278,10 @@
             }
           }
         },
-        "cryptosuite": "https://w3id.org/security#cryptosuite",
+        "cryptosuite": {
+          "@id": "https://w3id.org/security#cryptosuite",
+          "@type": "https://w3id.org/security#cryptosuiteString"
```

Comment:
This name makes me sad, I get what its doing, but it feels like the RDF version of:

```ts
type MyString = string;
const myString: MyString = "string";
```

instead of:

```ts
type CryptoString = "CRYDI2" | "CRYDI3" | "CRYDI5";
const cryptosuite: CryptoString = "CRYDI5";
```

Is there really no better option for this `@type` ?

Reply:
> Is there really no better option for this @type ?

At present, no, there isn't... because the range of cryptosuiteString over time can't be known (just like all future entries in the JOSE registry can't be known today).

Reply:
I agree with you, @OR13, that this is sub-optimal. However, I think it is unlikely to amount to much work for implementers, as they can hard-code the string values that their current or future implementation of the VC data model supports. The only other issue is the inability to shorten the string values into a vocabulary that can, say, be compressed with CBOR.

Reply:
im going to resolve this, since we don't seem to be requesting changes, but feel free to open again if you disagree.